### ISSUE TITLE
Pinch to Zoom not working in Ionic v1.0.0-rc.2

### ISSUE THREAD

**Issue by akreienbring** [JrSchild/jr-crop#13]
Hello,
I just discovered that by updating the Ionic lib to v1.0.0-rc.2 the pinch to zoom of your plugin stops working.
If I simply revert to v1.0.0-rc.1 pinch to zoom is working.

I have also created this issue on the Ionic project. 
https://github.com/driftyco/ionic/issues/3445

**Comment by perrygovier** [comment]
RC2 introduced a fix to stop preventDefault in certain gesture directions. When registering the gesture, it may help to add the direction where default should be disabled.
```
var options.prevent_default_directions = ['left','right'];
var contentTapGesture = $ionicGesture.on('tap', onContentTap, $element, options);
```

**Comment by akreienbring** [comment]
Yes, I saw that when I was inspecting the differences between rc.1 and rc.2.

Finally, this code fixes the issue for JrCrop:
```
      var options = {prevent_default_directions: ['left','right', 'up', 'down']};
      ionic.onGesture('touch transform drag dragstart dragend', function(e) {
    	  switch (e.type) {
          case 'touch':
            self.last_scale = self.scale;
            break;
          case 'drag':
            self.posX = self.last_posX + e.gesture.deltaX - transforming_correctX;
            self.posY = self.last_posY + e.gesture.deltaY - transforming_correctY;
            setPosWithinBoundaries();
            break;
          case 'transform':
            self.scale = Math.max(scaleMin, Math.min(self.last_scale * e.gesture.scale, scaleMax));
            setPosWithinBoundaries();
            break;
          case 'dragend':
            self.last_posX = self.posX;
            self.last_posY = self.posY;
            break;
          case 'dragstart':
            self.last_scale = self.scale;

            // After scaling, hammerjs needs time to reset the deltaX and deltaY values,
            // when the user drags the image before this is done the image will jump.
            // This is an attempt to fix that.
            if (e.gesture.deltaX > 1 || e.gesture.deltaX < -1) {
              transforming_correctX = e.gesture.deltaX;
              transforming_correctY = e.gesture.deltaY;
            } else {
              transforming_correctX = 0;
              transforming_correctY = 0;
            }
            break;
        }

        self.setImageTransform();

      }, self.options.modal.el, options);
    },
```
@JrSchild: It would be nice if you pull this in to your code...

**Comment by JrSchild** [comment]
Hi @akreienbring and @perrygovier. Thank you for looking into this. I will test it and put it in this weekend.

**Comment by JrSchild** [comment]
Released in version 1.0.0. This also introduces an incompatible API change. jrCrop is now it's own module, so please add it as a requirement.
